circle.yml

```diff
@@ -22,6 +21,6 @@ deployment:
     commands:
       - echo $gcloud_json_key_base64 | sed 's/ //g' | base64 -d > /tmp/gcloud_key.json
       - curl https://sdk.cloud.google.com | bash
+      - docker tag codeclimate/codeclimate $PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM
       - gcloud auth activate-service-account --key-file /tmp/gcloud_key.json
-      - gcloud docker -a
-      - docker push $PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM
+      - gcloud docker push $PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM
```

Comment:
There was a time when builder just ran `codeclimate init` using the cc binary. That may have used the docker image, so maybe this is vestigial to that?